docs/using_python_client_with_hazelcast_imdg.rst

```diff
@@ -1349,193 +1349,95 @@ the entry processor in the ``Map`` methods. See the following example.
 SQL
 ---
 
```

Comment:
maybe we can add a section corresponding to https://docs.hazelcast.com/hazelcast/5.0/sql/improving-performance

Reply:
OK, added that section. But 

```
To find out more about making sure certain map entries are stored on the same member, see Data Affinity.
```

part is not available in the Python client yet so I removed that part.